src/fairseq2/nn/utils/mask.py

```diff
@@ -171,3 +171,20 @@ def _generate_mask(indices: Tensor, max_row_len: int) -> Tensor:
     bool_mask = torch.full_like(float_mask, False, dtype=torch.bool)
 
     return bool_mask.scatter_(1, indices, True)
+
+
+def get_num_masks(mask: Tensor) -> Tensor:
```

Comment:
This is a convenience function which returns the number of masks present in a mask tensor of shape (bsz, seq_len). This is implemented in an efficient way, I believe this might be useful for some users.